index.js

```diff
@@ -0,0 +1,29 @@
+const whichTile = () => console.log('tile number');         //variable whichTile zostaje przypisany funkcji whichTile. 
```

Comment:
Change the functions name, `tileOnClickHandler` is much better giving picture what it's for